packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1688,21 +1695,39 @@ Dropdown.propTypes = {
   onHighlightedIndexChange: PropTypes.func,
   open: PropTypes.bool,
   placeholder: PropTypes.string,
-  position: PropTypes.oneOf(POSITIONS),
   renderItem: PropTypes.func,
   renderSelectedItem: PropTypes.func,
   search: PropTypes.oneOfType([PropTypes.bool, PropTypes.func]),
   searchQuery: PropTypes.string,
   searchInput: customPropTypes.itemShorthand,
   toggleIndicator: customPropTypes.shorthandAllowingChildren,
   triggerButton: customPropTypes.itemShorthand,
-  unstable_disableTether: PropTypes.oneOf([true, false, 'all']),
-  unstable_pinned: PropTypes.bool,
-  autoSize: PropTypes.oneOf<AutoSize>(AUTOSIZES),
   value: PropTypes.oneOfType([customPropTypes.itemShorthand, customPropTypes.collectionShorthand]),
   'aria-labelledby': PropTypes.string,
   'aria-invalid': PropTypes.bool,
   a11ySelectedItemsMessage: PropTypes.string,
+  // positioning props
+  align: PropTypes.oneOf(ALIGNMENTS),
```

Comment:
is there a common factory of popper PropTypes smwhere ? I don't know the answer, just checking